connectors/2.1/_index.md

```diff
@@ -1,11 +1,31 @@
 ---
-title: "Jakarta Connectors 2.1 (Under Development)"
-date: 2021-06-03
+title: "Jakarta Connectors 2.1"
+date: 2022-04-17
 summary: "Jakarta EE 10 Release"
 ---
 The Jakarta Connectors specification defines a standard architecture for Jakarta EE application components to connect to Enterprise Information Systems.
 .
 
+* [Jakarta Connectors 2.1 Release Record](https://projects.eclipse.org/projects/ee4j.jca/releases/2.1.0)
+* [Jakarta Connectors 2.1 Specification Document](./jakarta-connectors-spec-2.1.pdf) (PDF)
+* [Jakarta Connectors 2.1 Specification Document](./jakarta-connectors-spec-2.1.html) (HTML)
+* [Jakarta Connectors 2.1 Javadoc](./apidocs)
+* [Jakarta Connectors 2.1 XML Schema](https://jakarta.ee/xml/ns/jakartaee/connector_2_0.xsd)
+* [Jakarta Connectors 2.1 TCK](https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/staged/eftl/jakarta-connectors-tck-2.1.0.zip)
```

Comment:
This link must point to the final location the TCK will get after being promoted by the spec committee

https://download.eclipse.org/jakartaee/connectors/2.1//jakarta-connectors-tck-2.1.0.zip

Reply:
Also add links to sig, sha, and pub like this:

([sig](https://download.eclipse.org/jakartaee/connectors/2.1/jakarta-connectors-tck-2.0.0.zip.sig),  [sha](https://download.eclipse.org/jakartaee/connectors/2.1/jakarta-connectors-tck-2.0.0.zip.sha256),  [pub](https://jakarta.ee/specifications/jakartaee-spec-committee.pub))

Reply:
OK done